back/infolica/views/balance.py

```diff
@@ -154,12 +128,21 @@ def get_balance_files_view(request):
     files = []
     for filename in os.listdir(path):
         if filename.startswith(balance_filename_prefix) and (filename.endswith(".doc") or filename.endswith(".docx")):
-            files.append({'filename': filename, 'filepath': os.path.join(path, filename)})
-    
+            files.append({"filename": filename, "filepath": os.path.join(path, filename)})
+
     return files
 
 
-@view_config(route_name='balance_from_file', request_method='POST', renderer='json')
+def _formatNumeroBalance(cadastre_id, number):
```

Comment:
Shouldn't that not be camel cased?

Reply:
Done.
I'm not sure to understand when to use or not to use camel case. But for code uniformity, I totally agree 👍